mathics/builtin/layout.py

```diff
@@ -162,6 +172,8 @@ class Left(Builtin):
 
 class NonAssociative(Builtin):
     """
+    <url>:WMA link:https://reference.wolfram.com/language/ref/NonAssociative.html</url>
```

Comment:
This link is broken

Reply:
That is funny, but it is a builtin symbol in WMA, but does not have an entry in WR....